packages/js/src/redux/selectors/isWooSEO.js

```diff
@@ -9,11 +10,11 @@ import { getIsProduct } from "./editorContext";
 export const getIsWooSeoActive = () => Boolean( get( window, "wpseoScriptData.isWooCommerceSeoActive", false ) );
 
 /**
- * Checks if WooCommerce is active.
+ * Checks if WooCommerce is active. Used also in premium.
  *
  * @returns {boolean} True if WooCommerce is active.
  */
-export const getIsWooCommerceActive = () => Boolean( get( window, "wpseoScriptData.isWooCommerceActive", false ) );
+export const getIsWooCommerceActive = () => isWooCommerceActive();
```

Comment:
As mentioned before, no longer enforcing the `Boolean` part now. Which is a problem with the `"1"` or `""` that the script data will return due to the WP localize data first layer `(string)` coercion

Reply:
the script data will now return a boolean value